frontend/src/lib/lemon-ui/LemonSelect/LemonSelect.scss

```diff
@@ -30,4 +30,8 @@ body:not(.posthog-3000) {
             }
         }
     }
+
+    .LemonButton.LemonSelect {
+        --lemon-button-gap: 0.1875rem;
```

Comment:
The default is `0.25rem` so maybe this can be pushed to be even smaller. I'm going to be honest I can't see the difference